api/src/sconify/sconify.service.js

```diff
@@ -85,13 +93,23 @@ export async function sconify({
       );
     }
 
+    appEntrypoint = inspectResult.Config.Entrypoint?.join(' ') ?? null;
+    if (!appEntrypoint) {
+      throw new ForbiddenError(`Can't read entrypoint from docker image`);
+    }
+    logger.info({ appEntrypoint }, 'Entrypoint read from image.');
+
     // Pull the SCONE image
     logger.info('---------- 4 ---------- Pulling Scone image');
-    await pullSconeImage(SCONE_NODE_IMAGE);
+    if (configTemplate.SconeImage != '') {
```

Comment:
prefer `===` / `!==` to avoid JS wierdness

```
> 0 != ''
false
> 0 !== ''
true
```

```suggestion
    if (configTemplate.SconeImage !== '') {
```